lib.typ

```diff
@@ -15,22 +15,22 @@
 ///   theme: default_theme
```

Comment:
might be confusing if documentation isn't consistent, you should check for the rest of them too